### DISCUSSION TITLE
blog/laravel-filament

### DISCUSSION THREAD

**Discussion by giscus** [VernSG/yusufporto#5] [General]
# blog/laravel-filament

Panduan lengkap membuat CRUD dashboard dengan mudah menggunakan Laravel Filament.

https://www.yusufs.me/blog/laravel-filament

**Comment by Yu1chiro** [comment]
kelazz cik

**Reply by VernSG** [reply]
tq abanggg yui

**Comment by ItsAltoo** [comment]
ty bang sangat membantu 🙏

**Reply by VernSG** [reply]
sama" bang

**Comment by Vollyu** [comment]
Sosok Asli kink php cik

**Reply by VernSG** [reply]
alamak aku pemula banh
